apps/web/components/booking/BookingListItem.tsx

```diff
@@ -224,7 +224,7 @@ function BookingListItem(booking: BookingItemProps) {
   const showRecordingsButtons =
     (booking.location === "integrations:daily" || booking?.location?.trim() === "") && isPast && isConfirmed;
 
-  const title = decodeURIComponent(booking.title);
+  const title = booking.title;
```

Comment:
introduced in https://github.com/calcom/cal.com/pull/7490 to decode encoded booking title, throw this error if the title has malformed URI. so removed

![Screenshot 2023-03-13 at 21-16-42 Buchungen Cal com](https://user-images.githubusercontent.com/84864519/224761238-99acd6c1-1d11-4deb-b2ae-9c57d14f5cc5.png)